sql/hive/src/main/scala/org/apache/spark/sql/hive/client/ClientWrapper.scala

```diff
@@ -27,7 +27,7 @@ import scala.language.reflectiveCalls
 import org.apache.hadoop.fs.Path
 import org.apache.hadoop.hive.metastore.api.Database
 import org.apache.hadoop.hive.conf.HiveConf
-import org.apache.hadoop.hive.metastore.TableType
+import org.apache.hadoop.hive.metastore.{TableType => HTableType}
```

Comment:
It gets rid of a scalac warning (since `TableType` is also defined in the Spark side).